test/service/test_manifest_async.py

```diff
@@ -186,14 +186,16 @@ def test(self, mock_uuid, mock_helper):
                     path = '/manifest/files'
                     object_url = 'https://url.to.manifest?foo=bar'
                     object_key = 'some_object_key'
```

Comment:
The `object_key` variable in `TestManifestResponse.test_manifest` starts with `'manifests/'`. I would like that to be the same here unless that causes complications.